### PR TITLE
Bump Kommander chart

### PR DESCRIPTION
**Pull request by gracedo**
https://jira.d2iq.com/browse/D2IQ-64099
Bumps the `kommander` chart to pull in the latest changes which added the secret reloader annotations for karma and thanos proxies.

The following needs to be done (with #438 merge):
1. Bump the version of `mtls-proxy` used in `kommander-thanos` and `kommander-karma` and add the `secret.reloader.stakater.com/reload` annotations ✅ **DONE IN https://github.com/mesosphere/charts/pull/443**
1. Bump the deps of `kommander-thanos` and `kommander-karma` in `kommander` chart to pull in the latest changes from the above ✅ **DONE IN https://github.com/mesosphere/charts/pull/446**
1. Bump kommander version in `kubeaddons-kommander` ✅ **DONE IN THIS PR**

## Testing
I tested this manually by hosting `mtls-proxy`, `kommander-thanos`, `kommander-karma`, and `kommander charts` on my personal GH. I `konvoy up` on addons version `master`, then upgraded the kommander chart by deploying addons with my `kubeaddons-kommander` branch `gracedo/bug_certs_D2IQ-64099` (this branch has obviously now been updated to mesosphere's hosted kommander chart).
```
  - configRepository: https://github.com/mesosphere/kubeaddons-kommander
    #configVersion: master
    configVersion: gracedo/bug_certs_D2IQ-64099
    addonsList:
    - name: kommander
      enabled: true
```

Upon upgrading, I verified that updated secrets were federated out to the managed clusters (as they were before) then checked that the proxies now had the reloader annotation:
image
image

I checked Thanos and Karma UIs in the kommander cluster and saw that they were still able to connect to the proxies even after upgrading
image
image

### CODE REVIEW

**Comment by gracedo** [comment]
@juliangieseke @hectorj2f Can we make sure to use this updated kommander chart version for tomorrow's MWT?

File: addons/kommander/1.x/kommander-11.yaml [hunk 1]
```diff
@@ -0,0 +1,76 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: ClusterAddon
+metadata:
+  name: kommander
+  labels:
+    kubeaddons.mesosphere.io/name: kommander
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.0.0-10"
```
Comment (samvantran) [inline addons/kommander/1.x/kommander-11.yaml]: this should point to `-11`?

File: addons/kommander/1.x/kommander-11.yaml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+              secret.reloader.stakater.com/reload: "kubefed-admission-webhook-serving-cert"
+            webhook:
+              annotations:
+                secret.reloader.stakater.com/reload: "kubefed-admission-webhook-serving-cert"
+
+      kommander-karma:
+        karma:
+          deployment:
+            annotations:
+              configmap.reloader.stakater.com/reload: kommander-kubeaddons-config
```
Comment (samvantran) [inline addons/kommander/1.x/kommander-11.yaml]: `kommander-thanos` doesn't need the same annotation?

**Comment by gracedo** [comment]
@shaneutt Looks like that PR bumped to 0.4.12 which covers the 0.4.11 bump here :) closing this out!

**Pull request closed without merging**